packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js

```diff
@@ -1,65 +1,43 @@
-import { withFocusProps, withHoverProps } from '@klarna/higher-order-components'
-import { bool, func, node, object, shape, string } from 'prop-types'
+import { bool, func } from 'prop-types'
 import React from 'react'
-import styled, { withTheme } from 'styled-components'
+import { Button as GrommetButton } from 'grommet'
+import styled from 'styled-components'
+import { pxToRem } from '@zooniverse/react-components'
 
-const StyledButton = styled.button`
-  background: none;
-  border: 0;
-  cursor: ${props => props.disabled ? 'not-allowed' : 'pointer'};
-  opacity: ${props => props.disabled ? '0.3' : '1'};
-  padding: 0;
-`
-
-const Background = styled.circle`
-  fill: ${props => props.dark ? '#2d2d2d' : 'transparent'};
-  ${props => (props.hoveredOrFocused || props.active) && `
-    fill: #00979D;
-  `}
-  opacity: ${props => props.hoveredOrFocused ? '0.5' : '1'};
-`
+const StyledButton = styled(GrommetButton)`
+  background-color: ${props => props.active ? props.theme.global.colors.brand : (props.theme.dark ? props.theme.global.colors['dark-1'] : 'inherit')}; 
+  border-radius: 50%;
+  padding: ${pxToRem(13)};
 
-const IconSVG = styled.svg`
-  fill: ${props => props.dark ? 'white' : 'black'};
-  ${props => (!props.dark && (props.hoveredOrFocused || props.active)) && `
-    fill: white;
-  `}
-`
+  &:hover, &:focus {
+    background-color: ${props => props.theme.dark ? props.theme.global.colors['neutral-2'] : props.theme.global.colors['accent-2']};
 
-@withTheme
-@withHoverProps({ hovered: true })
-@withFocusProps({ focused: true })
-class Button extends React.Component {
-  getSize (sizeInPercent) {
-    const position = (100 - sizeInPercent) / 2
-    return {
-      x: `${position}%`,
-      y: `${position}%`,
-      height: `${sizeInPercent}%`,
-      width: `${sizeInPercent}%`
+    > svg {
+      fill: white;
     }
   }
 
+  > svg {
+    fill: ${props => props.active ? 'white' : (props.theme.dark ? 'white' : 'black')};
+    stroke: transparent;
+    width: ${pxToRem(18)};
```

Comment:
Height needs to be set as well here. Right now, the icons are inheriting the Grommet default for height now, 24px, and it's making the circle buttons oval now. Height should be the same as width to have the inner icon be square and the outer button be a circle.

Reply:
Yes it looked a little off thank you for catching!